CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ YOURLS Changelog
 _This file lists the main changes through all versions of YOURLS.  
 For a much more detailed list, simply refer to [commit messages](https://github.com/YOURLS/YOURLS/commits/master)._
 
+1.8.3
+---
+- improved: the API plugin -- more plugin functions (#3281), a sandbox and a plugin uninstall procedure (#3282)
+- improved: inline documentation, [online documentation](https://doc.yourls.org/) and unit tests
+- improved: concurrency during mass shortening (#3233)
+- improved: minor security fixes - sanitize step name during upgrade (#3055),
+    nonce on the logout link (#3264), salt cookie with newer hash (#3278),
+- improved: Remove ozh/phpass library and use password_* functions (#3232)
+- improved: third party binaries will now be automatically updated in the repository
```

Comment:
This is really nice for us, but can lead user to misinterpretation: third party binaries will _not_ be automatically updated from user POV.

Reply:
True. I'll mention this as a "nice to have for us" in the blog post, and point to [ozh/yourls-geoip-update](https://github.com/ozh/yourls-geoip-update) as a user friendly way to update anytime